docker/deployed/staging/api/config.json

```diff
@@ -41,6 +41,10 @@
       "KeepFiles": 5
     }
   },
+  "TelemetryPublisher": {
```

Comment:
I think we always include all the fields in the `.json` files, so maybe is worth adding `PublishinInterval` even with the default value? (If convinced, this also applies for testnet config json)